camel-frappe-api/src/main/java/com/ozonehis/camel/frappe/sdk/internal/operation/DefaultGetOperation.java

```diff
@@ -27,21 +28,12 @@ public DefaultGetOperation(
     protected FrappeResponse doExecute(HttpUrl httpUrl) {
         HttpUrl.Builder httpUrlBuilder = httpUrl.newBuilder();
         if (!fields.isEmpty()) {
-            StringBuilder fieldsAsString = new StringBuilder();
-            fieldsAsString.append("[");
-            for (int i = 0; i < fields.size(); i++) {
-                fieldsAsString.append("\"").append(fields.get(i)).append("\"");
-                if (i < fields.size() - 1) {
-                    fieldsAsString.append(", ");
-                }
-            }
-            fieldsAsString.append("]");
-            httpUrlBuilder.addQueryParameter("fields", fieldsAsString.toString());
+            String fieldsAsString =
+                    fields.stream().map(field -> "\"" + field + "\"").collect(Collectors.joining(", ", "[", "]"));
+            httpUrlBuilder.addQueryParameter("fields", fieldsAsString);
```

Comment:
Maybe we should add an example request and response for more clarity on how those actually look like.

Reply:
I'm hesitant to do that; We don't have yet the full request at this point. It only adds `fields` query param. So basically, the above code converts `["name", "age", "email"]` to `"[\"name\", \"age\", \"email\"]"`